Chaincase.UI/Pages/OverviewPage.razor

```diff
@@ -35,12 +35,31 @@
     <IonList>
         @foreach (var transaction in ViewModel.Transactions)
         {
-            <IonItemSliding>
-                <TransactionListItem TransactionViewModel="@transaction" />
-                <IonItemOptions side="end">
-                    <IonItemOption OnClick="@(() => OpenTransactionDetail(transaction))" color="secondary">Details</IonItemOption>
-                </IonItemOptions>
-            </IonItemSliding>
+            <IonItem button OnClick="@(() => OpenTransactionDetail(transaction))">
+                @if (!transaction.Confirmed)
+                {
+                    <IonIcon slot="start"
+                             color="medium"
+                             name="ellipsis-horizontal-circle-outline" />
+                }
+                else if (transaction.IsAmountPositive)
+                {
+                    <IonIcon slot="start"
+                             name="add-circle-outline"
+                             color="success" />
+                }
+                else
+                {
+                    <IonIcon slot="start"
+                             name="remove-circle-outline"
+                             color="danger" />
+                }
+                <IonLabel>
+                    <h3>@transaction.Label</h3>
+                    <h3 class="ion-text-end ion-float-end">@transaction.AmountBtc</h3>
```

Comment:
2 h3s have the same semantic precedence. For accessibility in history should contact label (clusters) come ahead of amounts?